WebClock/Models/ClockInOutRepository.cs

```diff
@@ -28,5 +28,11 @@ public IEnumerable<ClockInOut> GetClockOutsForThisMonth(int id)
                 .Where(x => x.Type == ClockType.Out)
                 .Where(date => date.Date.Month == DateTime.Now.Month);
         }
+
+        public IEnumerable<ClockInOut> GetClockOutsForAllTimeById(int id)
+        {
+            return _repository
```

Comment:
To kwestia indywidualnych upodobań ale tak krótkich linii wydaje się nie ma potrzeby łamać